CMakeLists.txt

```diff
@@ -101,6 +102,9 @@ add_definitions(${GTCLANG_EXTERNAL_DEFINITIONS})
 # Output summary of the packages
 mchbuild_report_result("Package summary" ${package_info})
 
+# Support of RPATH-exports of dawn?

```

Comment:
remove `?` and maybe add a 
```
if(NOT(DEFINED DAWN_RPATH_DIR))
   message(FATAL_ERROR "DAWN_RPATH_DIR is not exported by DAWN")
endif()
```
would ensure that dawn is properly exporting the variable